script/Arcueid/Day 11/11_00_ARC11_3C.txt

```diff
@@ -397,12 +397,12 @@ Even though she promised we'd stick together,
 [sha:0fcefa3f7a23e15feaf438fff221503cd2f6a9fe]{
 -- Page 315, Offset 30445.
 -- どうして――――どうしてまたひとりに戻るっていうんだ、おまえは――――
-Why―――Why are you going back to being alone―――
+Why―――Why are you going back to being alone―――?
 }
 [sha:366943de91155e5522554065b4fd790b5b7b12a7]{
 -- Page 316, Offset 30446.
 -- 「―――なんでなんだ、アルクェイド―――！！」
-"Why the hell, Arcueid―――!"
+"Why the hell, Arcueid―――?!"
```

Comment:
is this intoned as a question in va?

Reply:
hard to tell since he just yells it out, I'll revert